gptme/prompts.py

```diff
@@ -103,6 +103,17 @@ def prompt_gptme(interactive: bool) -> Generator[Message, None, None]:
 Use the patch tool to edit existing files, or the save tool to overwrite.
 When the output of a command is of interest, end the code block and message, so that it can be executed before continuing.
 
+When using nested codeblocks (codeblocks within codeblocks), always specify a language tag for both the outer and inner blocks.
+This is critical for correct parsing - without language tags, nested blocks may be cut off prematurely.
+For example:
+```python
+print("This is the outer block")
+# Inner block needs a language tag:
+```shell
+echo "This is the inner block"
+```
+```
```

Comment:
I'm not sure, but I've had a hunch and maybe even the experience that this prompt makes LLMs more prone to outputting incorrectly nested toolcalls, such as:

```shell
cat ...

```save test.txt
asd
```
```

Which of course isn't valid, because a markdown block isn't valid bash.